book/modules/appendix1-exercises.tex

```diff
@@ -52,59 +94,253 @@
+							number solutions.) Therefore, Kokoro's second
+							configuration is possible.
+
+						\item For the third configuration, let $x$ be the number
+							of chickens, and let $y$ be the number of rabbits.
+							Using the information given in the problem, we have
+							\[
+								\systeme{x+y=42, y=77}.
+							\]
+							Row reduce this system, and we get
+							\[
+								\systeme{x+y=42, y=-\frac{7}{2}}.
+							\]
+							This system is consistent, and the complete solution
+							is given by
+							\[
+								\left\{
+								\begin{array}
+									{ccc} x & = & \frac{91}{2} \\ y & = & -\frac{7}{2}
+								\end{array}\right.
+							\]
+							However, this is not a natural number solution. Since
+							this is the only solution to the system, we conclude
+							that there's no natural number solution to the
+							system. Therefore, Kokoro's third configuration is
+							not possible.
+					\end{enumerate}
+			\end{enumerate}
+		\end{solution}
+
+		\prob For each statement below, determine whether it is true or false.
+		Justify your answer.
 		\begin{enumerate}
 			\item A system of linear equations of 4 variables with 3 equations is
 				always consistent.
 
 			\item Any system of linear equation with $0x_{1}+0x_{2}+\cdots+0x_{n}
 				=0$ being one of the equations must be consistent.
 
-			\item There are $m,c\in \R$ so that the $y$-axis is the solution set to the equation $y=mx+c$.
-			
-			\item There are $m,c\in \R$ so that the $x$-axis is the solution set to the equation $y=mx+c$.
-			
-			\item There are $m_1,m_2,c\in \R$ so that the $x$-axis (in $\R^3$) is the solution set to the equation $z=m_1x+m_2y+c$.
+			\item There are $m,c\in \R$ so that the $y$-axis is the solution set
+				to the equation $y=mx+c$.
+
+			\item There are $m,c\in \R$ so that the $x$-axis is the solution set
+				to the equation $y=mx+c$.
+
+			\item There are $m_{1},m_{2},c\in \R$ so that the $x$-axis (in
+				$\R^{3}$) is the solution set to the equation $z=m_{1}x+m_{2}y+c$.
 
 			\item A system of exactly one equation can have an empty solution set.
 		\end{enumerate}
+		\begin{solution}
+			\begin{enumerate}
+				\item False. A counterexample is given by
+					\[
+						\systeme{x_1+x_2+x_3+x_4=1, x_1+x_2+x_3+x_4=2, x_1+x_2+x_3+x_4=3}.
+					\]
+
+				\item False. A counterexample is given by
+					\[
+						\systeme{0x_1+0x_2=0, 0x_1+0x_2=1}.
+					\]
+
+				\item False. Reason: The $y$-axis can be described as $\Set*{\mat{0 \\ y}\in\R^2: y\in\R}$.
+					If there exists $m,c$ so that the $y$-axis is the solution
+					set to the equation $y = mx + c$, then both $x=0, y=1$ and $x
+					=0, y=2$ are solutions to the equation. Plug in $x=0, y=1$, and
+					we get $m=0$, $c=1$. Plug in $x=0$, $y=2$, and we get $m=0$,
+					$c=2$. Of course, $c=1$ and $c=2$ cannot happen at the same
+					time. Therefore, there's no $m,c\in \R$ so that the $y$-axis
+					is the solution set to the equation $y = mx + c$.
```

Comment:
This is way too long. something like: Assume the $y$ axis can be represented as the complete solution to $y=mx+b$ for some $m,b$. Since `(x,y)=(0,0)` and `(x,y)=(0,1)` are both on the $y$ axis, we know ...